assets/src/scss/layout/_block-classes.scss

```diff
@@ -1,26 +1,79 @@
 .wp-block-query-pagination {
+  a,
+  .page-numbers {
+    color: $black;
+    padding: $sp-1 $sp-1x;
+    user-select: none;
+    border: 1px solid transparent;
+    border-radius: 4px;
+  }
+
   a {
-    color: black;
-    font-weight: bold;
+    font-weight: 500;
```

Comment:
This rule doesn't apply to the current item because it's not a link, but according to the mockups I think that it should have the same font weight?